src/main/resources/markdown/releases.md

```diff
@@ -38,7 +49,7 @@ Apache ZooKeeper 3.7.0 is our current release, and 3.6.3 our latest stable relea
 
 [Apache ZooKeeper 3.6.3 Source Release](https://www.apache.org/dyn/closer.lua/zookeeper/zookeeper-3.6.3/apache-zookeeper-3.6.3.tar.gz)([asc](https://downloads.apache.org/zookeeper/zookeeper-3.6.3/apache-zookeeper-3.6.3.tar.gz.asc), [sha512](https://downloads.apache.org/zookeeper/zookeeper-3.6.3/apache-zookeeper-3.6.3.tar.gz.sha512))
 
-### Apache ZooKeeper 3.5.9
+### Apache ZooKeeper 3.5.9 (maintenance)
```

Comment:
I think it would be nice to actually stating the EoL date here, like 

> (maintenance, expected to be EoL from ...)

we mentioned "1st of June 2022" in the mail thread which seems reasonable, giving us plenty of time to communicate this and make a "final" 3.5 release after 3.8 is shipped

Reply:
This is done, thanks.